src/docs/adv/cli.rst

```diff
@@ -0,0 +1,43 @@
+Command Line Interface
+======================
+
+NetsBlox includes a command line tool providing basic management and troubleshooting functionality at ``./bin/netsblox``.
```

Comment:
Just noticing this so making a note here: this also needs to be updated to the current CLI.